docs/hardware/devices/rak/peripherals.mdx

```diff
@@ -20,34 +20,52 @@ values={[
 ]}>
 <TabItem value="GPS">
 
+## GPS Modules
+
+### RAK5005-0 / RAK1910
+
 To add a GPS to the RAK5005-O base board, you need the [RAK1910 GPS sensor](https://store.rakwireless.com/collections/wisblock-sensor/products/rak1910-max-7q-gnss-location-sensor). It is supported on slot A of the 5005 board via UART.
 
 - uBlox MAX-7Q GPS module
 - GPS and GLONASS satellite support
 
-Further information on the RAK1910 can be found on the [RAK Documentation Center](https://docs.rakwireless.com/Product-Categories/WisBlock/RAK1910/Overview/#product-description).
-
-US Distributor - Purchase link: [Rokland](https://store.rokland.com/products/rak-wireless-rak1910-wisblock-gnss-location-module)
+### RAK19003 / RAK12500
 
 To add a GPS to the RAK19003 base board, you need the [RAK12500 GPS sensor](https://store.rakwireless.com/products/wisblock-gnss-location-module-rak12500). It is supported via I<sup>2</sup>C on slot B for firmware versions 1.49 and above.
 
 - uBlox Zoe-M8Q GNSS receiver
 - GPS, GLONASS, QZSS and BeiDou satellite support
 
-Further information on the RAK12500 can be found on the [RAK Documentation Center](https://docs.rakwireless.com/Product-Categories/WisBlock/RAK12500/Overview/#product-description).
-
-US Distributor - Purchase link: [Rokland](https://store.rokland.com/products/rak-wireless-rak12500-gnss-gps-location-module-u-blox-zoe-m8q)
+### Resources
+- RAK Documentation Center
+  - [RAK12500](https://docs.rakwireless.com/Product-Categories/WisBlock/RAK12500/Overview/#product-description)
+  - [RAK 1910](https://docs.rakwireless.com/Product-Categories/WisBlock/RAK1910/Overview/#product-description)
```

Comment:
I hate being so picky but there's an extra space "RAK1910"

Its funny because the content of this page is wrong, but it's the extra space that bugs. me.